data/json/recipes/weapon/bashing.json

```diff
@@ -246,11 +246,12 @@
   {
     "type": "recipe",
     "activity_level": "LIGHT_EXERCISE",
-    "result": "knuckle_steel",
+    "result": "knuckle_scrap",
```

Comment:
The old recipe id needs to be added to `data/json/obsolete.json` to avoid error message in old saves.